pages/linux/wikit.md

```diff
@@ -0,0 +1,20 @@
+# wikit
+
+> A command line program for getting Wikipedia summaries easily.
+> More information: <https://github.com/KorySchneider/wikit>.
+
+- Show a short introduction of Linux on Wikipedia:
+
+`wikit linux`
+
+- Specify a language:
+
+`wikit linux -l {{lang_code}}`
+
+- Open the full Wikipedia article in the default browser:
+
+`wikit linux -b`
+
+- Open a disambiguation CLI menu:
+
+`wikit linux -d`
```

Comment:
```suggestion
- Show a short summary of a specific topic on Wikipedia:

`wikit {{topic}}`

- Specify a [l]anguage:

`wikit {{topic}} --lang {{language_code}}`

- Open the full Wikipedia article in the default [b]rowser:

`wikit {{topic}} --browser`

- Open a disambiguation CLI menu:

`wikit {{topic}} -d`
```

Reply:
> 

Thanks for your suggestions, long option is more readable! But one thing is the "--browser" option is not the same with "-b", "-b" would use the default browser to open up Wikipedia page without assigning a {{browser_name}}. So I think I would like to leave the "-b" option in this page since it is easier.

Reply:
> > 
> 
> Thanks for your suggestions, long option is more readable! But one thing is the "--browser" option is not the same with "-b", "-b" would use the default browser to open up Wikipedia page without assigning a {{browser_name}}. So I think I would like to leave the "-b" option in this page since it is easier and commonly used.

Agreed, feel free to revert it back to `-b`. With it addressed, this PR is GTG.